kuma/wiki/tasks.py

```diff
@@ -298,6 +298,17 @@ def build_locale_sitemap(locale):
         os.makedirs(directory)
 
     queryset = Document.objects.filter_for_list(locale=locale)
+    # Exclude any document whose slug starts with one of these prefixes:
+    #     - "Experiment:":
+    #         these documents are disallowed by robots.txt
+    #     - "Mozilla/QA/Marionette/WebDriver/status":
+    #         this document is a hard-wired redirect to an external domain, so
+    #         it can't be excluded as the content-based redirects are by the
+    #         "filter_for_list" method above
+    for slug_prefix in ('Experiment:',
+                        'Mozilla/QA/Marionette/WebDriver/status'):
```

Comment:
I wish there was a different way to do this. It seems ugly to special-case a single URL. Maybe we should remove the external redirect and instead have content that says "Moved to ``<bugzilla link>``"? But it works, so 🤷‍♂️

Reply:
I feel exactly the same, I don't like it. I initially thought I could change this into a content-based redirect before realizing that's not possible for external redirects.

Reply:
Since the [entire set of Marionette-related documents that are now hard-redirected](https://github.com/mozilla/kuma/blob/c724ba2e3db2a47806410731639e308cd0da2636/kuma/redirects/redirects.py#L836) should probably be ignored in general, perhaps we should either add them to the default filters of `filter_for_list` or soft-delete them?